griptape/templates/api/extensions/chat_gpt_plugin_manifest.json.j2

```diff
@@ -0,0 +1,18 @@
+{
+  "schema_version": "v1",
+  "name_for_human": "{{ name_for_human }}",
+  "name_for_model": "{{ name_for_model }}",
+  "description_for_human": "{{ description_for_human }}",
+  "description_for_model": "{{ description_for_model }}",
+  "auth": {
+    "type": "none"
+  },
+  "api": {
+    "type": "openapi",
+    "url": "{{ api_url }}",
+    "is_user_authenticated": false
```

Comment:
Side note: maybe we can use ChatGpt Plugins as a source of inspiration for how we can do our own Tool authentication.